textile/features.textile

```diff
@@ -115,9 +115,9 @@ h3(#restclient). RestClient
 *** @(RSC15b1)@ @ClientOptions#restHost@ has not been set to an explicit value (see "RSC11a":#RSC11a) and @ClientOptions#port@ is not set and @ClientOptions#tlsPort@ is not set
 *** @(RSC15b2)@ An array of @ClientOptions#fallbackHosts@ is provided
 *** @(RSC15b3)@ the deprecated @ClientOptions#fallbackHostsUseDefault@ option is set to @true@
-** @(RSC15k)@ When none of the statements in "RSC15b":#RSC15b are true then host fallback does not apply, and failing HTTP requests that would have "qualified for a retry against a fallback host (see RSC15d)":#RSC15d will instead result in an error immediately
+** @(RSC15k)@ When none of the statements in "RSC15b":#RSC15b are true then host fallback does not apply, and failing HTTP requests that would have "qualified for a retry against a fallback host (see RSC15l)":#RSC15l will instead result in an error immediately
```

Comment:
This is an interesting case of spec point mutation that I hadn't considered when working on https://github.com/ably/specification/pull/120. However, I still think it's too much of a change to this spec point to permit modification like this as it does imply change required for SDK implementations relating directly to this spec point when the change they need to act upon is only the replacement of `RSC15d` with `RSC15l`.

Therefore I think the change here could be much simpler - being no change at all. Because we keep spec points in place in the case of _Replacement_ and the replacement text includes a pointer to the replacement spec point, then I would prefer to see `RSC15d` to stay here (and for `RSC15a` too). That makes this diff smaller and, I argue, the change easier to reason about and understand when looking back in future.

Reply:
:+1: reverted that change